AsyncDisplayKit/Details/ASPINRemoteImageDownloader.h

```diff
@@ -9,8 +9,12 @@
 #import <Foundation/Foundation.h>
 #import "ASImageProtocols.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 @interface ASPINRemoteImageDownloader : NSObject <ASImageCacheProtocol, ASImageDownloaderProtocol>
 
-+ (instancetype)sharedDownloader;
++ (ASPINRemoteImageDownloader *)sharedDownloader;
```

Comment:
It can't be instancetype, because if I subclass, then the return type of `+[TRPINRemoteImageDownloader sharedDownloader]` is `TRPINRemoteImageDownloader` but what I'll get is actually an `ASPINRemoteImageDownloader`.